<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifier_ab</class>
 <widget class="QDialog" name="modifier_ab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier abonnee</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/New folder/favicon.ico</normaloff>../../Desktop/New folder/favicon.ico</iconset>
  </property>
  <widget class="QLineEdit" name="ID_ABMOD">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NOM_MOD">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>NOM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PRENOM_MOD">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="MOD">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>ID_AB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PRENOM</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AGE_MOD">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
